cmd/otelcontribcol/go.mod

```diff
@@ -8,6 +8,8 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/countconnector v0.97.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector v0.97.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/exceptionsconnector v0.97.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/connector/failoverconnector v0.97.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/connector/routingconnector v0.97.0
```

Comment:
```suggestion
```
The routing connector is already added as required a few lines below.

Reply:
Ah yep thanks that shouldn’t be there. Will ipdate